test/fixture/registration/fixture.go

```diff
@@ -24,3 +29,11 @@ func GetRegistrationEntries() []*common.RegistrationEntry {
 
 	return []*common.RegistrationEntry{blogEntry, databaseEntry}
 }
+
+func FromFile(fileName string) []*common.RegistrationEntry {
```

Comment:
Good opportunity to move this to `test/util`? :)

Reply:
Tricky one, I wasn't sure since it's not general enough, currently we need to convert to the right type at the end 
```
dat, _ := ioutil.ReadFile(path)
json.Unmarshal(dat, &regEntries)
return regEntries.Entries
```
So It would be more like GetRegistrationEntries, which already exists. Clearly, this needs a better approach. Let me see what I can come up with, I don't want fixtures getting out of control.

Reply:
We can introduce new packages under `test/util/` to keep things tidy, like `test/util/registration` or something